specs-go/config.go

```diff
@@ -90,8 +98,10 @@ type Mount struct {
 	// Source specifies the source path of the mount.  In the case of bind mounts on
 	// Linux based systems this would be the file on the host.
 	Source string `json:"source"`
-	// Options are fstab style mount options.
-	Options []string `json:"options,omitempty"`
+	// Options are fstab style mount options. (this field is platform dependent)
+	Options []string `json:"options,omitempty" platform:"linux,solaris"`
+	// Readonly specifies if the mount should be read-only. (this field is platform dependent)
+	Readonly bool `json:"readonly" platform:"windows"`
```

Comment:
On Wed, Jun 22, 2016 at 07:57:58PM -0700, Mrunal Patel wrote:
> > -	// Options are fstab style mount options.
> > -	Options []string `json:"options,omitempty"`
> > +	// Options are fstab style mount options. (this field is platform dependent)
> > +	Options []string `json:"options,omitempty" platform:"linux,solaris"`
> > +	// Readonly specifies if the mount should be read-only. (this field is platform dependent)
> > +	Readonly bool `json:"readonly" platform:"windows"`
>
> Could you parse a constant `readonly` from existing `options` to set
> this boolean for Windows?

Or stay compatible with Linux by using ‘ro’ and ‘rw’ [1].  Matching
Linux isn't a big deal, but I'm surprised there's no Windows analog to
[1].  Is “readonly” really the only knob?

[1]: http://man7.org/linux/man-pages/man8/mount.8.html#FILESYSTEM-INDEPENDENT_MOUNT%20OPTIONS